src/pages/trails/ballots/components/BallotChip.vue

```diff
@@ -26,6 +26,8 @@ q-chip(square text-color="white" :class="`${type}-chip`").capitalize.no-margin.t
 </template>
 
 <style lang="sass">
+.ballot-type.voted-chip
+  background: #00A402
```

Comment:
move hex to gobal css variable to keep track of the colors we're using throughout the app